test/cosmoz-bottom-bar.html

```diff
@@ -205,6 +214,25 @@
 
 			});
 		});
+
+		suite('height management', function () {
+			var heightManagementParent, bottomBar;
+
+			setup(function () {
+				heightManagementParent = fixture('heightManagement');
+				bottomBar = Polymer.dom(heightManagementParent).querySelector('cosmoz-bottom-bar');
+			});
+
+			test('setting matchParent to true should set bottom-bar height to the height of the parent', function () {
+				bottomBar.matchParent = true;
+				assert.equal(bottomBar.offsetHeight, 100);
+			});
+
+			test('setting matchParent to false should set bottom-bar less than to the height of the parent', function () {
+				bottomBar.matchParent = false;
+				assert.isBelow(bottomBar.offsetHeight, 100);
```

Comment:
But thats not happening according to logic.That's the bug or problem in documentation?